app/controllers/upvotes_controller.rb

```diff
@@ -7,27 +7,34 @@ def create
         review_id: @review.id,
         user_id: current_user.id
       )
-      @review.score += 1
-
-      if @upvote.save && @review.save
-        flash[:notice] = "Upvote created successfully."
-        redirect_to @beach
-      else
-        flash[:notice] = "Upvote not created successfully."
-        render :new
+      respond_to do |format|
+        if @upvote.save
+          format.html { redirect_to :back,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.